themes/Flex/templates/base.html

```diff
@@ -107,19 +107,19 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
               {% set PAGES_SORT_ATTRIBUTE = 'title' %}
           {%- endif %}
           {% for page in pages|sort(attribute=PAGES_SORT_ATTRIBUTE) %}
-          <li><a href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}">{{ page.title }}</a></li>
+          <li><a href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}" aria-label="{{ page.title }}">{{ page.title }}</a></li>
```

Comment:
You don't need the `aria-label` here because there's already text content in the link